### DISCUSSION TITLE
Create a branch from an issue, auto suggests branch name

### DISCUSSION THREAD

**Discussion by pigam** [magit/forge#483] [Ideas]
We use the following workflow in my team:
1. create an issue, e.g. `#2 Add this wonderful feature` to describe what to do
2. in a personal fork, create a branch named after the issue : `2-add-this-wonderful-feature`.
3. create a PR/MR from the issue branch 

What I would like:
- from the issue list or the issue view, if I create a new branch, auto propose a branch name derived from issue title.

This functionality already exists at least in gitlab: from the issue view, if you click on the «Create MR Button» you can opt to create a branch instead.
![image](https://user-images.githubusercontent.com/38874936/163142180-6547d8eb-850d-494a-b467-9fcff0bb2f8a.png)

**Comment by tarsius** [comment]
Thanks for the suggestion. Unfortunately there's quite a backlog already. (Also spring is finally here.) I don't think I will get around to look into this request any time soon.

**Comment by tarsius** [comment]
The suggested feature sounds useful if you happen to name branches like this (which I don't), but IMO this crosses the line to feature creep, so I won't add it to Forge.

You might be able to implement this yourself, if it is a big enough time saver to be worth it.
